### DISCUSSION TITLE
What year did the Titanic sink in the Atlantic Ocean on 15 April?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#338] [Galaxy Trivia] [answered]
**Hint:** -111

**Comment by GameExplorer** [comment]
1912

**Comment by thuongtruong1009** [comment, marked as answer]
The Titanic sank in the North Atlantic Ocean on April 15, 1912, after striking an iceberg during its maiden voyage from Southampton, England, to New York City, USA. The sinking of the Titanic was one of the most tragic maritime disasters in history, resulting in the loss of over 1,500 lives. The incident prompted significant changes in maritime safety regulations and the development of new technologies and procedures to prevent similar disasters in the future.

**Comment by peterbaccaro** [comment]
The Titanic sank in the Atlantic Ocean on 15 April 1912. The ship, which was on its maiden voyage from Southampton, England, to New York City, struck an iceberg late on the night of 14 April and sank in the early hours of 15 April.
